op-e2e/actions/l2_batcher_test.go

```diff
@@ -76,7 +76,6 @@ func NormalBatcher(gt *testing.T, deltaTimeOffset *hexutil.Uint64) {
 	p := &e2eutils.TestParams{
 		MaxSequencerDrift:   20, // larger than L1 block time we simulate in this test (12)
 		SequencerWindowSize: 24,
-		ChannelTimeout:      20,
```

Comment:
Interesting that none of these action tests that set a lower channel timeout actually seem to depend on that value at all...

Reply:
indeed. the tests only require that the data fits in the channel.